Portfolio/src/views/HomeView.vue

```diff
@@ -1,43 +1,44 @@
 <script setup lang="ts">
+import Wrapper from "@/Wrapper.vue";
 import Copyright from "../components/Copyright.vue"
 import { homeViewContent } from "./home-view-content";
 
 
-
 </script>
 
 <template>
-    <!-- default layout -->
-    <header class="home-wrapper">
-        <div class="homepage-leftcontainer">
-            <h2 class="home-smallName">{{ homeViewContent.firstName }}</h2>
-            <h1 class="homepage-title">{{ homeViewContent.titleCareer }}</h1>
-        </div>
-        <div class="homepage-rightcontainer">
-            <h1 class="home-firstName">{{ homeViewContent.firstName }}</h1>
-            <h1 class="home-lastName">{{ homeViewContent.lastName }}</h1>
-        </div>
-    </header>
-    <!-- default layout -->
+    <Wrapper>
+
+        <!-- default layout -->
+        <header class="home-wrapper">
+            <div class="homepage-leftcontainer">
+                <h2 class="home-smallName">{{ homeViewContent.firstName }}</h2>
+                <h1 class="homepage-title">{{ homeViewContent.titleCareer }}</h1>
```

Comment:
You have a mix of naming conventions here.
`leftcontainer` and `smallName` is the mix im referring to. No casing in the first name and camelCase in the other. Sticking with one convention will make your css and html cleaner.

I would recommend:

`home-page-left-container`
`home-page-small-name`